### ISSUE TITLE
Why is the F1 score plummeting in epoch 1?

### ISSUE THREAD

**Issue by lczhang06** [shmsw25/single-hop-rc#3]
I ran this training code and the F1 score of the last time prediction on epoch 0 was 66.290, however, all the F1 score during epoch 1  were just between 1 to 2.  What could possibly be the cause?
![TIM截图20190815151733](https://user-images.githubusercontent.com/29545448/63079000-d2c65880-bf6f-11e9-9126-66c7e0c58ba5.png)

**Comment by shmsw25** [comment]
Hi @lczhang06.
Thanks for your interest. When finetuning BERT, the model can be unstable when it is about to saturate and give inf loss. It seems like your model reaches to 66.290 F1 score then becomes unstable. You can interpret this result as the best performance of 66.290. This number is slightly lower than the number reported in the paper --- difference in batch_size, etc can contribute to the marginal difference in performance.

**Comment by jellying** [comment]
> I ran this training code and the F1 score of the last time prediction on epoch 0 was 66.290, however, all the F1 score during epoch 1 were just between 1 to 2. What could possibly be the cause?
> ![TIM截图20190815151733](https://user-images.githubusercontent.com/29545448/63079000-d2c65880-bf6f-11e9-9126-66c7e0c58ba5.png)

Hi @shmsw25 , I want to reimplement the 66 F1 result on --task hotpot-all, but I just get 56  F1. Did you run the script in default setting? If not, could you please share me your settings?

**Comment by shmsw25** [comment]
⬆️  #4
